### ISSUE TITLE
Reference documentation is down

### ISSUE THREAD

**Issue by DSpeichert** [php-opencloud/openstack#141]
http://refdocs.os.php-opencloud.com/ hasn't been working for over a week now.

**Comment by jamiehannaford** [comment]
Yeah it hasn't been working in a while now. I do not have the bandwidth right now to fix and come up with a long-term hosted solution.

**Comment by laszlof** [comment]
@jamiehannaford what exactly do you need spec wise?

**Comment by jamiehannaford** [comment]
@laszlof We need a box (either VM or container) that can:

1. present rendered HTML over port 80
2. listen on a specific port and rebuild docs when a PR happens

**Comment by laszlof** [comment]
@jamiehannaford let me know the exactly details on what needs to be setup, and how the builds get triggered. If it doesn't take too much time on our part, we (nexcess.net) can set something up to host them.

**Comment by laszlof** [comment]
@jamiehannaford any update on this?

**Comment by laszlof** [comment]
Great. Thanks. I'll throw something together later today and will get back with you on getting the webhook setup.

**Comment by laszlof** [comment]
@jamiehannaford I have the environment setup. Shoot me your email address and I'll send you the IP for DNS update and webhook POST URL.

**Comment by jamiehannaford** [comment]
@laszlof Awesome. My email is jamie.hannaford@rackspace.com

**Comment by laszlof** [comment]
Email sent. Please note that https://github.com/php-opencloud/openstack/pull/151 will need to be merged before doc building will work.

**Comment by laszlof** [comment]
Refdocs are now available at: http://refdocs.os.php-opencloud.com/ 

This issue can be closed.
